src/Domains/Social/DataTransferObject/MessagesNotificationsPayloadDto.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Social\DataTransferObject;
+
+use Spatie\LaravelData\Data;
+
+class MessagesNotificationsPayloadDto extends Data
```

Comment:
@rwhite27 since this is a msg move it here https://github.com/bakaphp/kanvas-ecosystem-api/tree/add-dto-messages-notification-payload/src/Domains/Social/Messages/DataTransferObject